### PR TITLE
added tagline to home page

### PR DESCRIPTION
**Pull request by aquidam**
Added tagline on the home page. 

image

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <h1>Welcome to my awesome website !</h1>
+<h2>It's <span style="color: red;">amazing!!!</span></h2>
```
Comment (nadiaauger) [inline app/views/pages/home.html.erb]: As we saw together we don't want the amazing word in our homepage. Can you change it ?

**Review by nadiaauger — CHANGES_REQUESTED** [review]
Please change it @aquidam 👍

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (aquidam) [reply app/views/pages/home.html.erb]: @nadiaauger What do you want the new word to be?

Comment (nadiaauger) [reply app/views/pages/home.html.erb]: Awesome please

Comment (nadiaauger) [reply app/views/pages/home.html.erb]: `<h2>It's <span style="color: red;">awesome!!!</span></h2>`

Comment (aquidam) [reply app/views/pages/home.html.erb]: @nadiaauger fixed!

**Review by nadiaauger — APPROVED** [review]
LGTM !!!!

**Pull request merged**